Feat/add skip dirs option (#33)
aquasecurity/trivy-action: README.md

| `ignore-unfixed` | Boolean | false                              | Ignore unpatched/unfixed vulnerabilities      |
| `vuln-type`      | String  | `os,library`                       | Vulnerability types (os,library)              |
| `severity`       | String  | `UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL` | Severities of vulnerabilities to be displayed |

> | `skip-dirs`       | String  |                                   | Specify the directory where the traversal is skipped |

Reviewer: ```suggestion
| `skip-dirs`       | String  |                                   | Comma separated list of directories where traversal is skipped |
```
